.eslintrc.json

```diff
@@ -8,7 +8,8 @@
     ],
     "globals": {
         "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
+        "SharedArrayBuffer": "readonly",
+        "__DEV__": "readonly"
```

Comment:
what is this doing?

Reply:
Expo defines this global to signal whether you are developing locally. `environment.ts` uses this to set the environment stage. This change specifies `__DEV__` as a global so that eslint won't complain about it.